ansible/monitoring.yml

```diff
@@ -30,6 +30,7 @@
       set_fact:
         env: "{{ instance_tags['tags']['env'] | default('unknown') }}"
         color: "{{ instance_tags['tags']['color'] | default('unknown') }}"
+        kind: "{{ instance_tags['tags']['kind'] | default('peer') }}"
```

Comment:
Why not explicitly tagging non-static AWS instances as `peer` (as done for `seed` [here](https://github.com/aeternity/infrastructure/pull/84/files#diff-b1a76c7f484f0748d6486ec8e6b80adaR40)), and placing default `unknown` here? Anyway debatable so ok.

Reply:
We still need a default to run the playbook on bare-metal for example